src/components/CDate/CDate.js

```diff
@@ -127,6 +141,7 @@ export default {
           on: {
             input(value) {
               self.value = value;
+              self.sendToEventBus('Changed', { value });
```

Comment:
Also, could you merge `Changed` and `Cleared` as that seems like a same event with different value (null if cleared).